sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -15689,4 +15688,56 @@ public void testStringOperationsNullableInference()
               )
       );
   }
+
+  @SqlTestFrameworkConfig(numMergeBuffers = 4)
+  @Test
+  public void testGroupingSetsWithAggrgateCase()
```

Comment:
I thought I could use this in checking the results (in case of normal & decoupled strategies - as both use the same tests).
iq test would be checking the plan & in a specified strategy (In this case it doesn't verify the results the decouple mode)